Doc/whatsnew/3.10.rst

```diff
@@ -93,6 +93,8 @@ Other Language Changes
   :meth:`~object.__index__` method).
   (Contributed by Serhiy Storchaka in :issue:`37999`.)
 
+* XXX Describe changes to NamedTemporaryFile functionality. Re 2020-09-28-04-56-04.bpo-14243.YECnxv.rst
```

Comment:
Needs to move to `3.12.rst`, and can be about half the length of what's currently in the NEWS file

Reply:
Implemented. Whats new is moved to 3.12.rst and only 2 lines now